python/dataproc_templates/hive/util/hive_ddl_extractor.py

```diff
@@ -0,0 +1,170 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import argparse
+import pprint
+import sys
+import io
+from logging import Logger
+from typing import Any, Dict, Optional, Sequence
+
+import dataproc_templates.util.template_constants as constants
+import pandas as pd
+from dataproc_templates import BaseTemplate
+from google.cloud import storage
+import pyspark
+from pyspark.sql import DataFrameWriter, SparkSession
+from google.cloud import bigquery
+from google.cloud.bigquery.table import Table
+from google.cloud.bigquery.dataset import Dataset
+from datetime import datetime
+
+
+def WriteToCloud (ddls_to_rdd, bucket, path):
+    # Write
+    now = datetime.now()
+    timedate = now.strftime("%m-%d-%Y %H.%M.%S")
+    ddls_to_rdd.coalesce(1).saveAsTextFile("gs://"+bucket+"/"+path+timedate)
```

Comment:
The timedata subpath can be kept, but the concatenation with "gs://" would be removed here